lib/pbench/cli/server/shell.py

```diff
@@ -15,7 +16,17 @@ def app():
     except (ConfigFileNotSpecified, BadConfig) as e:
         print(e)
         sys.exit(1)
-    return create_app(server_config)
+
+    try:
+        # Initialize the pbench databases
+        db_session = init_db(server_config)
+    except Exception as e:
+        print(
+            f"Some Error occurred while initializing the database engine: ERROR: {e.__traceback__}"
```

Comment:
Not sure it is worth catching all exceptions and minimizing them.  Perhaps we can just let the exception be raised so we get the full stack trace and normal python exception handling.

If we come across known, common, errors which produce a stack trace for which we don't need the stack trace, then we can catch them to quiet the noise.